code/modules/mob/living/emote.dm

```diff
@@ -232,6 +232,16 @@
 	message = "jumps!"
 	hands_use_check = TRUE
 
+/datum/emote/living/jump/run_emote(mob/living/user, params, type_override, intentional)
```

Comment:
This one's... I dunno. it's not *actually* jumping, and animated emotes like this rub me the wrong way a bit. Is the same one from tg?

Reply:
i believe so? it's also fine as its just some extra flavor to an emote

Reply:
ahem.
spinflipclapspinflipclap

Animated emotes are kinda. Meh. Especially since we have pixelshifting.

Reply:
we kill people for spin flip clapping, again this is just extra flavor to the default emotes

Reply:
The emote spam isn't really an issue for us. I'm just skeptical of emotes that move you like that in general.